src/caselawclient/Client.py

```diff
@@ -743,6 +748,7 @@ def eval(
             url=self._path_to_request_url(path),
             headers=headers,
             data=data,
+            timeout=(CONNECT_TIMEOUT, READ_TIMEOUT),
```

Comment:
Do we use eval for write operations which may potentially take longer? Is it worth making these parameters which just default to the constants?